docs/quickstart/fan_out_replication.rst

```diff
@@ -0,0 +1,99 @@
+.. include:: ../global.rst.inc
+
+.. _quickstart-fan-out-replication:
+
+Fan-out replication
+===================
+
+This quick-start example demonstrates how to implement a fan-out replication setup where datasets on a server (A) are replicated to multiple targets (B, C, etc.).
```

Comment:
There should be a reference to the section that you added to `docs/configuration/overview.rst`. Search for `:ref:` anywhere in the docs. And maybe throw in a back-reference to this quick-start example as well :)